app/forms/hyrax/forms/pcdm_collection_form.rb

```diff
@@ -6,6 +6,10 @@ module Forms
     # @api public
     # @see https://github.com/samvera/valkyrie/wiki/ChangeSets-and-Dirty-Tracking
     class PcdmCollectionForm < Valkyrie::ChangeSet # rubocop:disable Metrics/ClassLength
+      include Hyrax::FormFields(:core_metadata)
+      include Hyrax::FormFields(:collection_basic_metadata)
+      include Hyrax::FormFields(:collection_metadata)
+
       property :title, required: true, primary: true
```

Comment:
I'm wondering if title should be defined as a property here since core_metadata is included.